source/android/Renaissance/lib/http_services.dart

```diff
@@ -186,48 +166,79 @@ class HttpService {
     }
   }
 
+  Future<User> createPath(
+      String title,
+      String description,
+      List<Map<String, String>> milestones,
+      String? photo,
+      List<Map<String, Object>> topics) async {
+    String url = baseUrl + '/path/create-path/';
+    var body = jsonEncode({
+      'title': title,
+      'description': description,
+      'milestones': milestones,
+      'topics': topics,
+      'photo': photo,
+    });
+    //log(body);
+    log(HttpHeaders.authorizationHeader);
+    Response res = await post(Uri.parse(url), headers: headers, body: body);
+    if (res.statusCode == 200) {
+      return User.fromJson(jsonDecode(res.body));
+    } else if (res.statusCode == 403) {
+      //await refreshToken();
+      throw Exception("403.");
```

Comment:
Throwing exception is a powerful, yet dangerous approach. You should handle the exception and print appropriate error on page rather than whole exception

Reply:
I have improved the feedback text. Thanks for the notification!